src/main/java/io/vlingo/wire/fdx/inbound/rsocket/RSocketChannelInboundReader.java

```diff
@@ -117,7 +112,7 @@ private SocketAcceptorImpl(final ChannelMessageDispatcher dispatcher, final Stri
             rawMessageBuilder.workBuffer().put(payloadData);
             
             dispatcher.dispatchMessagesFor(rawMessageBuilder);
-          } catch (final Throwable t) {
+          } catch (final Exception t) {
```

Comment:
Not catching all exceptions will stop the inbound channel. Instead we should ignore the message and continue processing